src/java/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "java",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Why do we need to bump this given that we are not making any Java Feature changes?